moveit_core/collision_detection_bullet/CMakeLists.txt

```diff
@@ -66,21 +66,21 @@ if(BUILD_TESTING)
   ament_add_gtest(test_bullet_collision_detection test/test_bullet_collision_detection_pr2.cpp)
   target_link_libraries(test_bullet_collision_detection moveit_test_utils ${MOVEIT_LIB_NAME})
   # TODO: remove if transition to gtest's new API TYPED_TEST_SUITE_P is finished
-  if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  if(CMAKE_CXX_COMPILER_ID MATCHES "(GNU|Clang)")
```

Comment:
wouldn't a generator expression be more idiomatic CMake nowadays?

Reply:
Good idea, I'll take a look at it. Thanks!

Reply:
I don't believe it would be more idiomatic nor do I believe it's even possible. There is no generic regex match generator expression last time I checked.

Reply:
No, you're right there does not seem a regex match generator impression. Maybe it would have worked with a boolean variable which is then used as condition in the generator expression, but I like your proposal to refactor this into a "global" setting to avoid copy&pastes better.